src/main/java/seedu/taassist/model/moduleclass/ModuleClass.java

```diff
@@ -48,12 +46,27 @@ public ModuleClass(String className) {
      * @param sessions A list of sessions.
      */
     public ModuleClass(String className, List<Session> sessions) {
+        requireAllNonNull(className, sessions);
+        checkArgument(isValidModuleClassName(className), MESSAGE_CONSTRAINTS);
+        this.className = className;
+        this.sessions = new UniqueList<>();
+        this.sessions.addAll(sessions);
```

Comment:
After the above change:
```suggestion
        this.sessions.setElements(sessions);
```